index.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: custom
+---
+**OpenScanHub** is a service for static and dynamic analysis. By default it uses `Cppcheck`, `ShellCheck`, the static analyzers embedded in GCC and Clang, `find-unicode-control`, and the `Gitleaks` tool. Other tools for static and dynamic analysis can be enabled on demand while submitting an OpenScanHub scan.
+
+By default it can analyze RPM packages, tarballs and OpenShift containers.
```

Comment:
`OpenScanHub can analyze RPM packages and source code tarballs.`  Scanning of OpenShift containers is powered by `cspodman`, which is not publicly available yet.